extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -33,35 +37,58 @@ public function index() {
 		$user = FacebookClient::getInstance()->getWikiaUser( $fbUserId );
 
 		if ( ( $user instanceof User ) && ( $fbUserId !== 0 ) ) {
-			$this->errorMsg = '';
+			$errorMsg = '';
```

Comment:
Setting all the response data in one go in each conditional, so can't set a single val here.